extensions/wikia/WikiaApi/WikiaApiQueryDomains.php

```diff
@@ -29,6 +29,7 @@ protected function getDB() {
 	public function execute() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this function "execute" is 22 which is greater than 20 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)